### DISCUSSION TITLE
Prevent focused tests

### DISCUSSION THREAD

**Discussion by thabalija** [infinum/js-linters#51] [General]
We should prevent commiting fdescribe/fit.

```js
{
  "overrides": [
    {
      "plugins": ["jasmine"],
      "rules": ["jasmine/no-focused-tests": 2]
    }
  ]
}
```

Jasmine plugin: [eslint-plugin-jasmine](https://github.com/tlvince/eslint-plugin-jasmine)

**Comment by fvoska** [comment]
I agree with adding the rule, this will prevent us from committing code that has focused tests.

Just as a side-note, projects should also have code coverage gates, no matter how high or low the current percentages are, just have gates slightly below current metrics. The build should fail if those gates do not pass. That way, even if you do not have this rule, the tests will almost certainly fail if you have focused some individual test (suite) and the build will not pass.

@thabalija or @danipavic can one of you create a PR?

**Reply by thabalija** [reply]
I will create PR 👍

**Reply by thabalija** [reply]
Opened https://github.com/infinum/js-linters/pull/56
